package/MDAnalysis/coordinates/TRZ.py

```diff
@@ -358,13 +358,21 @@ def _seek(self, nframes):
 
         .. versionadded:: 0.9.0
         """
-        maxi_l = long(maxint)
-
-        framesize = long(self._dtype.itemsize)
-        seeksize = framesize * nframes
+        # On python 2, seek has issues with long int. This is solve in python 3
+        # where there is no longer a distinction between int and long int.
+        if six.PY2:
```

Comment:
I guess a Mock could do. But it would have to fail where an actual large file would fail, and it would have to capture how the behaviour differs among different versions of python. This would be tricky.